src/pages/ingredient-category/ingredient-category-page.ui.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+import { INGREDIENT_CATEGORIES } from '@/widgets/category-ingredient-container/category-ingredient-container.config';
+import { IngredientCategoryContainer } from '@/widgets/category-ingredient-container/category-ingredient-container.ui';
+
+export function IngredientCategoryPage() {
+  return (
+    <div className="container mx-auto p-6">
+      <h1 className="mb-4 text-2xl font-bold">재료 카테고리</h1>
+      <p className="mb-6 text-gray-600">원하시는 재료 카테고리를 선택해주세요</p>
+
+      <IngredientCategoryContainer ingredientCategories={INGREDIENT_CATEGORIES} />
+    </div>
```

Comment:
이 부분도 아래와 같이 수정하면 될 것 같습니다.

```suggestion
      <IngredientCategoryContainer categories={INGREDIENT_CATEGORIES} />
    </div>
```